lib/resources/example_json/r4_examples_communication.rb

```diff
@@ -0,0 +1,98 @@
+module Cerner
+  module Resources
+
+    R4_COMMUNICATION_ENTRY ||= {
+      "resourceType": "Communication",
+      "id": "C-1123_0",
+      "meta": {
+          "versionId": "0",
+          "lastUpdated": "2020-07-16T21:31:35.457Z"
+      },
+      "text": {
+          "status": "generated",
+          "div": "<div xmlns=\"http://www.w3.org/1999/xhtml\"><p><b>Communication</b></p></div>"
+      },
+      "extension": [
+        {
+          "valueCodeableConcept": {
+            "coding": [
+              {
+                "system": "http://hl7.org/fhir/task-status",
+                "code": "completed",
+                "display": "Completed"
+              }
+            ],
+            "text": "Completed"
+          },
+          "url": "https://fhir-ehr.cerner.com/r4/StructureDefinition/email-status"
+        }
+      ],
+      "status": "completed",
+      "category": [
+        {
+          "coding": [
+            {
+              "system": "http://terminology.hl7.org/CodeSystem/communication-category",
+              "code": "notification",
+              "display": "Notification"
+            }
+          ],
+          "text": "Notification"
+        }
+      ],
+      "priority": "urgent",
+      "subject": {
+        "reference": "Patient/12505984",
+        "display": "JOHNSON, MARIE"
+      },
+      "topic": {
+        "text": "CDI Query"
```

Comment:
I think we should change the topic text to something other than CDI Query since that is a different category.  Maybe Stacey can suggest something real.

Reply:
Talked with Stacey about this last night, and she said topic is supposed to be the subject of the message, and consumers will be able to send us anything want but 'CDI Query' is the example text we have been using with Nuance.